.gitignore

```diff
@@ -9,9 +9,10 @@ docs/demos/test-program.cpp
 include/awkward/kernels.h
 src/awkward/_kernel_signatures.py
 src/awkward/_kernel_signatures_cuda.py
+src/awkward/_v2/_connect/cuda/_kernel_signatures.py
 
 # Generated CUDA kernels
-src/cuda-kernels/awkward_*.cu
+# src/cuda-kernels/awkward_*.cu
```

Comment:
It can be deleted, rather than commented out. Git is providing the history of it having been there. (It will be easier to find it in the blame log if it's one change, existence → non-existence, than two: existence → commented out → non-existence.)